src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,28 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+
```

Comment:
Redundant line spacing here :P Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.

Reply:
Same for redundant spacing below in the rest of your code